save-preprocessor/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ configureSpringBoot()
 dependencies {
     implementation(project(":save-common"))
     implementation("org.eclipse.jgit:org.eclipse.jgit:${Versions.jgit}")
+    runtimeOnly("org.jetbrains.kotlin:kotlin-reflect:1.4.31")
```

Comment:
Why?

Reply:
`@ConstructorBinding` and `@ConfigurationProperties` require it.

Reply:
I don't quite see why they should require it. Maybe you should add `@ConstructorBinding` on constructor and not on the class?

Reply:
Nevermind, I believe it's correct, because it needs to aces setters based on property name known only at runtime.